lisk/package.json

```diff
@@ -24,13 +24,15 @@
 		"node": "10.15.3",
 		"npm": "6.4.1"
 	},
-	"main": "src/index.js",
+	"main": "dist/index.js",
 	"scripts": {
-		"start": "node src/index.js",
+		"start": "node dist/index.js",
```

Comment:
[pm2-lisk.json(https://github.com/LiskHQ/lisk-sdk/blob/3391-typescript-lisk-core/lisk/build/target/etc/pm2-lisk.json#L5), [Dockerfile](https://github.com/LiskHQ/lisk-sdk/blob/3391-typescript-lisk-core/lisk/Dockerfile#L48) and [docker-compose.yml](https://github.com/LiskHQ/lisk-sdk/blob/3391-typescript-lisk-core/lisk/docker/docker-compose.yml#L19) must be updated also.